packages/@glimmer/node/lib/serialize-builder.ts

```diff
@@ -96,13 +96,13 @@ class SerializeBuilder extends NewElementBuilder implements ElementBuilder {
   pushRemoteElement(
     element: SimpleElement,
     cursorId: string,
-    nextSibling: Option<SimpleNode> = null
+    _insertBefore: Option<null>
```

Comment:
I'd prefer to continue passing `insertBefore` around (even though we assert on values other than `null` or `false`).

```suggestion
    insertBefore: Option<null>
```